frontend/app/src/components/dashboard/summary-card/SummaryCard.vue

```diff
@@ -28,10 +28,10 @@ const slots = useSlots();
   <VCard
     :loading="isLoading"
     :class="`dashboard__summary-card__${name}`"
-    class="pb-1"
+    class="pb-4"
```

Comment:
Also here do you think it's possible to migrate to RuiCard?

Reply:
updated